<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>701</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDockWidget" name="consoleWidget">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="floating">
           <bool>false</bool>
          </property>
          <property name="allowedAreas">
           <set>Qt::BottomDockWidgetArea</set>
          </property>
          <widget class="QWidget" name="dockWidgetContents_2">
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="restoreButton">
            <property name="text">
             <string>Restore</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelkp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLoadTool"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelkp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadTool"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionLoadTool">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/document_open.png</normaloff>:/icons/icons/document_open.png</iconset>
   </property>
   <property name="text">
    <string>Load tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/help_about.png</normaloff>:/icons/icons/help_about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/document_save.png</normaloff>:/icons/icons/document_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icons/icons/document_save_as.png</normaloff>:/icons/icons/document_save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
